src/lib/Scenes/Show/Components/ShowHeader/ShowHeader.tsx

```diff
@@ -132,8 +132,11 @@ export class ShowHeader extends React.Component<Props, State> {
   render() {
     const { isFollowedSaving } = this.state
     const {
-      show: { artists, images, is_followed, name, partner, exhibition_period },
+      show: { artists, images, is_followed, name, partner, exhibition_period, followedArtists },
     } = this.props
+    const fairfollowedArtistList =
+      (followedArtists && followedArtists.edges && followedArtists.edges.map(fa => fa.node.artist)) || []
+    const uniqArtistList = [...new Set(fairfollowedArtistList.concat(artists))]
```

Comment:
this will remove the order, so you'll probably want something like `_.uniq`

Reply:
good call! i'll update